payment_ippay/views/payment_view.xml

```diff
@@ -0,0 +1,19 @@
+<odoo>
+
+    <record id='acquirer_form_ippay' model='ir.ui.view'>
+        <field name='name'>acquirer.form.ippay></field>
+        <field name='model'>payment.acquirer</field>
+        <field name='inherit_id' ref='payment.acquirer_form'/>
+        <field name='arch' type='xml'>
+            <xpath expr="//group[@name='acquirer']" position="inside">
+                <group>
+                    <field name='api_url' widget="url" attrs="{'invisible': [('provider','!=', 'ippay')],
+                    'required': [('provider','=', 'ippay')]}"/>
```

Comment:
required attribute already taken care in python field declaration, is this repetition ?